week3.js

```diff
@@ -0,0 +1,223 @@
+	const result = 'a ' + color + ' car';
+	return result;
+}
+
+
+
+//3
+function info(person = {firstName:"mark", lastName:"trump", age:50, eyeColor:"blue"}) {
+	return person;
+}
+
+
+
+//4
+function vehicleType(color, code){
+  if(code === 1){
+  return console.log("a " + color + " car")}
+  else if(code === 2){
+  	return console.log("a " + color + " motorbike")}
+}
+
+
+
+//5
+3 === 3 ? console.log("yes") : console.log("no");
+
+
+
+//6
+function vehicle(color, code, age){
+  if(code === 1){
+  	   if(age > 0){
+  	   	console.log("a " + color + " used car");
+  	   }
+  	   	else{console.log("a " + color + " car");}
+  	   }
+  else if(code === 2){
+  	   if(age > 0){
+  		console.log("a " + color + " used motorbike");
+  	   }
+  		else{
+  	   	console.log("a " + color + " motorbike");}
+  	   }
+}
+
+
+
+//7
+// method 1: using objects (adding another layer of objects for fun!)
+vehicle = {
+	"motor": {
+		id: "motorbike",
+    	id2: "bus",
+     	id3: "truck",
+     	id4: "car"
+     },
+     "railed": {
+     	id: "train",
+     	id2: "tram",
+     },
+     "aircraft": {
+     	id: "airplane",
+     	id2: "helicopter",
+     }
+}
+const firstKey = Object.keys(vehicle)[0];
+const secondKey = Object.keys(vehicle)[1];
+const thirdKey = Object.keys(vehicle)[2];
+//----------------------------------------------------
+console.log(vehicle["aircraft"]["id2"]);
+//logs helicopter
+console.log(vehicle[thirdKey]["id2"]);
+//logs helicopter
+//----------------------------------------------------
+
+//method 2: using arrays
+const vehicles = ["motorbike", "caravan", "bike", "bus"];
+
+
+
+
+//8
+console.log(vehicles[2]);
+
+
+
+
+//9
+const vehicles = ["motorbike", "caravan", "bike", "bus"];
+
+
+function vehicle(color,code,age){
+  let i;
+  if(code === 1){
+  	i = vehicles[0];
+  }
+  else if(code === 2){
+  	i = vehicles[1];
+  }
+  else if(code === 3){
+  	i = vehicles[2]
+  } 
+  else if(code === 4){
+  	i = vehicles[3]
+  } 
+
+  let x;
+  if(age <= 1){
+  x = "new";
+  }
+  else{
+	x = "used";
+  }
+   console.log("a",color,x,i);
+}
+vehicle("green", 3, 1)
+
+/////////////////////////////////////////////////////////
+//10
+const vehicles = ["car", "motorbike", "caravan", "bike"];
+
+let vehicleToService="";
+for(let counter = 0; counter < vehicles.length; counter ++){
+  if(counter === vehicles.length - 1){
+  	vehicleToService += vehicles[counter] + "s.";
+  } 
+  else {vehicleToService += vehicles[counter] + "s, ";}
+ }
+
+ console.log(" \"Amazing Joe's Garage, we service" , vehicleToService,"\"");
+
+
+
+//11
+const vehicles = ["car", "motorbike", "caravan", "bike"];
+vehicles.push("plane");
+console.log(vehicles);
+let vehicleToService="";
+for(let counter = 0; counter < vehicles.length; counter ++){
+  if(counter === vehicles.length - 1){
+  	vehicleToService += vehicles[counter] + "s.";
+  } 
+  else {vehicleToService += vehicles[counter] + "s, ";}
+ }
+
+console.log(" \"Amazing Joe's Garage, we service" , vehicleToService,"\"");
```

Comment:
There is no need for the slashes here `console.log("Amazing Joe's Garage, we service" , vehicleToService);`, this works fine. You could even use backticks which makes it even easier to work with variables and strings, see https://developer.mozilla.org/en/docs/Web/JavaScript/Reference/Template_literals